Azure/azure-cli
# [Storage] Fix #22236: `az storage entity insert`: Fix `--if-exists fail` not working

**Related command**

`az storage entity insert`

**Description**

Fix #22236 

**Testing Guide**

First of all insert an entity
```
az storage entity insert -e PartitionKey=AAA RowKey=BBB Content=test -t $table_name --account-name $account_name
```

Then insert the same entity with `--if-exists fail`
```
az storage entity insert -e PartitionKey=AAA RowKey=BBB Content=test --if-exists fail -t $table_name --account-name $account_name
```

---

This checklist is used to make sure that common guidelines for a pull request are followed.
